jobs-api-server/src/test/java/org/zowe/jobs/services/zosmf/ModifyJobZosmfRequestRunnerTest.java

```diff
@@ -5,7 +5,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2019
+ * Copyright IBM Corporation 2020
```

Comment:
Looks like a find replace gone wrong - should go to 2019, 2020

Reply:
Changed to range